promptRemove/kubevirt.io.virtualmachine.vue

```diff
@@ -0,0 +1,178 @@
+<script>
+import { mapState, mapGetters } from 'vuex';
+import { HCI } from '@/config/types';
+import { isEmpty } from '@/utils/object';
+import Parse from 'url-parse';
+
+export default {
+  name: 'PromptRemove',
```

Comment:
This one definitely needs an update

Reply:
@richard-cox  sorry, I don't understand the meaning of the phrase?  Is it changing promptRemove to a more specific name?

Reply:
@WuJun2016 Correct